src/platforms/Token/RemoveTokenRewardConfirmation.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  Alert,
+  AlertIcon,
+  Collapse,
+  HStack,
+  Stack,
+  Text,
+  useDisclosure,
+} from "@chakra-ui/react"
+import useRemoveGuildPlatform from "components/[guild]/AccessHub/hooks/useRemoveGuildPlatform"
+import Button from "components/common/Button"
+import ConfirmationAlert from "components/create-guild/Requirements/components/ConfirmationAlert"
+import WithdrawPoolModal from "./WithdrawPoolModal"
+import usePool from "./hooks/usePool"
+
+const RemoveTokenRewardConfirmation = ({ isOpen, onClose, guildPlatform }) => {
+  const { onSubmit, isLoading } = useRemoveGuildPlatform(guildPlatform.id)
+
+  const { data: poolData, refetch } = usePool(
+    guildPlatform.platformGuildData.chain,
+    BigInt(guildPlatform.platformGuildData.poolId)
+  )
+  const [, , , poolBalance] = poolData || []
+
+  const {
+    isOpen: withdrawIsOpen,
+    onOpen: withdrawOnOpen,
+    onClose: withdrawOnClose,
+  } = useDisclosure()
+
+  return (
+    <>
+      <ConfirmationAlert
+        isLoading={isLoading}
+        isOpen={isOpen}
+        onClose={onClose}
+        onConfirm={onSubmit}
+        title="Remove reward"
+        description={
+          <>
+            <Collapse in={!!poolBalance}>
+              <Alert status="warning">
+                <Stack>
+                  <HStack gap={0}>
+                    <AlertIcon mt={0} />{" "}
+                    <Text fontWeight={"bold"}>Funds remaining in pool</Text>
+                  </HStack>{" "}
+                  <Text>
+                    Any funds not withdrawn will be permanently lost after deletion.
```

Comment:
Admins will be able to withdraw on Etherscan, so they won't be lost, we'll need to think about a better copy